co-log-benchmark/Setup.hs

```diff
@@ -0,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain
```

Comment:
I think that `Setup.hs` is not needed for `cabal`, it works without this file pretty fine

Reply:
that require you to have cabal-install. But with Setup.hs, you may install library without `cabal-install` that what most of the distributions do.

Reply:
@qnikst Could you, please, elaborate on this? How is it possible to install Haskell library without `cabal-install` and why one might want to do this?

Reply:
you can always run:

```
ghc Setup.hs
./Setup configure
./Setup build
./Setup install
```
It will not install any dependency, but you don't need any additional tool to do that. Source distribution like `Gentoo`, `NixOS` do that, and other when you prepare a binary package. Because dependencies and package management is solved by the special tools.